### PR TITLE
bugfix - KITOSUDV-2099 anvender snitflade UI fix

### PR DESCRIPTION
**Pull request by JacobMalling**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/app/components/it-system/it-interface/it-interface-catalog.controller.ts [hunk 1]
```diff
@@ -656,13 +656,11 @@
...
         public showOrganizationNames(interfaceName: string, interfaceId: number) {
             this.itInterfaceId = interfaceId;
-            this.usedByOrganizationNamesGrid.dataSource.fetch(() => {
-                this.usedByOrganizationNamesModal.setOptions({
-                    close: (_) => true,
-                    resizable: false,
-                    title: `Organisationer der anvender snitfladen: ${interfaceName}`
-                });
-                this.usedByOrganizationNamesModal.center().open();
+            this.usedByOrganizationNamesGrid.dataSource.read();
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/it-system/it-interface/it-interface-catalog.controller.ts:659]: det skal ikke først være efter "then" som før?

**Review by mrjsawdk — APPROVED** [review]
et enkelt spørgsmål

File: Presentation.Web/app/components/it-system/it-interface/it-interface-catalog.controller.ts (hunk 1 above)
Comment (mrjsawdk) [reply Presentation.Web/app/components/it-system/it-interface/it-interface-catalog.controller.ts:659]: Det var mere om det burde være sådan

Comment (JacobMalling) [reply Presentation.Web/app/components/it-system/it-interface/it-interface-catalog.controller.ts:659]: Der er ikke callback på samme måde i read som der er i fetch. Og den del med setOptions bruges primært til at sætte titlen på modalen, hvor alt vi skal bruge allerede er tilgængeligt

**Pull request merged**